src/app/home/home.component.scss

```diff
@@ -7,3 +7,25 @@ q {
   font-size: 1.2rem;
   quotes: "« " " »";
 }
+.card-body {
+  background-color: #6c5ce7;
+  border-radius: 10px;
+  height: 194px;
+}
+h1 {
```

Comment:
This will change the color of all `<h1>` elements in the component. 
It works, but it is preferrable to use a class instead of the element directly. You can add the class you need by changing the html to `<h1 class="your-class">` and using the class selector like you did for the others

Reply:
Okay I'll fix it and make a new PR